tests/clang_parser.cpp

```diff
@@ -7,20 +7,20 @@ namespace bpftrace {
 namespace test {
 namespace clang_parser {
 
-void parse(const std::string &input, BPFtrace &bpftrace)
+static void parse(const std::string &input, BPFtrace &bpftrace, int result)
```

Comment:
```suggestion
static void parse(const std::string &input, BPFtrace &bpftrace, int result = 0)
```
This would just mean fewer changes to the rest of the code and mean new tests wouldn't have to worry about the return result unless it was important

Reply:
aaah, right.. I keep forgetting about this ;-) thanks